### ISSUE TITLE
Add size badge

### ISSUE THREAD

**Issue by kevinbarabash** [Khan/KaTeX#1016]
https://github.com/ngryman/badge-size

We'd want to show both minified and gzipped size.

**Comment by santos22** [comment]
HI @kevinbarabash! Which file would you like to show the size of?

**Comment by kevinbarabash** [comment]
@santos22 I was thinking katex.min.js.

**Comment by edemaine** [comment]
No, it's not.  It's in the release.  Is there an NPM-based badge tool?

**Comment by daniellimws** [comment]
The closest I could find is 
![npm bundle size (minified + gzip)](https://img.shields.io/bundlephobia/minzip/katex.svg)
which gives the size of the minified and gzipped npm bundle.

**Comment by ylemkimon** [comment]
@daniellimws Actually, tagged (release) commit has it, e.g., https://github.com/Khan/KaTeX/tree/v0.9.0/dist. And you can display the size of `katex.min.js.`, for example, like: ![](http://img.badgesize.io/Khan/KaTeX/v0.9.0/dist/katex.min.js) `![](http://img.badgesize.io/Khan/KaTeX/v0.9.0/dist/katex.min.js)`.

**Comment by edemaine** [comment]
Oh, it's on master too -- presumably from the last release.  I stand corrected!

**Comment by daniellimws** [comment]
Ok so we'll need to remember to be updating the size badge after every release.

**Comment by edemaine** [comment]
@daniellimws No, we shouldn't have to remember anything. It could be part of the release script, though.

**Comment by daniellimws** [comment]
Oh cool I'll make a PR for this some time tomorrow, while updating the release script.
